### PR TITLE
daily processing pipeline

### PR DESCRIPTION
**Pull request by nicHoch**
create a second processing pipeline to handle not TM packets triggered processing steps  

this pipeline runs on a cron bases (once a day?) and performs product creations (fits files)

first included processing is HK-L+ to ANC-ASPECT files

### CODE REVIEW

File: stixcore/io/ProcessingHistoryStorage.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            #             level TEXT NOT NULL,
+            #             type TEXT NOT NULL,
+            #             fitspath TEXT NOT NULL,
+            #             p_date FLOAT NOT NULL
+            #         )
+            # ''')
+            # self.cur.execute('''CREATE INDEX if not exists processed_flare_products_idx ON
+            #                     processed_flare_products (flareid, flarelist, version, name,
+            # level, type)''')
+
```
Comment (samaloney) [inline stixcore/io/ProcessingHistoryStorage.py:49]: Does this need to be uncommented?

File: stixcore/io/FlareListManager.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+if 'pytest' in sys.modules:
+    # only set the global in test scenario
+    from stixcore.data.test import test_data
+    SDCFlareListManager.instance = SDCFlareListManager(test_data.rid_lut.RID_LUT, update=False)
+
+
+if __name__ == '__main__':
+    flm = SDCFlareListManager(Path("./flarelist.scv"), update=False)
+    print(flm)
+    print(flm.flarelist)
```
Comment (samaloney) [inline stixcore/io/FlareListManager.py:139]:
```suggestion
```

File: setup.cfg [hunk 3]
```diff
@@ -30,6 +30,7 @@ install_requires =
     sunpy>=4.0.1
     watchdog
     requests
+    stixdcpy @ git+ssh://git@github.com/i4ds/stixdcpy.git
```
Comment (samaloney) [inline setup.cfg:33]: Is there a tag or version we can set here? This will just install what ever is on master

**Review by samaloney — APPROVED** [review]
Looks ok to me.

I'm concerned about out use of context mangers and multiple threads - I don't think they are natively thread aware. Might not be an issue for this PR but brought to my attention based on our conversation the other day and this PR

**Comment by samaloney** [comment]
Also I see the tests are failing?

File: stixcore/io/ProcessingHistoryStorage.py (hunk 1 above)
Comment (nicHoch) [reply stixcore/io/ProcessingHistoryStorage.py:49]: yes as mentioned i removed/uncomment some parts that not yet are necessary but already be prepared for images on flarelist entries

**Pull request merged**